icon_validator/rules/workflow_validators/workflow_id_validator.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+from typing import List
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+from icon_validator.workflow.unmarshal import read_workflow
+from icon_validator.workflow.model import Workflow
+from icon_validator.styling import YELLOW
+
+class WorkflowIDValidator(KomandPluginValidator):
+    def id_exists(self, workflow: Workflow):
+        if workflow.kom.workflowVersions[0].id:
```

Comment:
This seems like a duplicate of the one above, did it not get changed in a copy/paste?